### PR TITLE
BitType: Fix #216 by setting entitiesPerPixel to 1/64

### LINKED ISSUE #216: Default construction of an ArrayImg<BitType> creates overly large storage arrays
See https://github.com/gab1one/imglib2-bittype for a demonstration of this issues, which lets an `ArrayImg<BitType>` to take up as much memory as an `ArrayImg<LongType>`.

### PR DESCRIPTION
**Pull request by gab1one**

### CODE REVIEW

File: src/main/java/net/imglib2/type/logic/BitType.java [hunk 1]
```diff
@@ -326,7 +326,7 @@ public String toString()
 	@Override
 	public Fraction getEntitiesPerPixel()
 	{
-		return new Fraction();
+		return new Fraction(1, 64);
```
Comment (ctrueden) [inline src/main/java/net/imglib2/type/logic/BitType.java]: Awesome catch!

Could you please add a regression test that calls `ArrayImgs.bits` and then rips out the `long[]` (via `getCurrentStorageArray()`) and asserts that it is the right length?

Comment (gab1one) [reply src/main/java/net/imglib2/type/logic/BitType.java]: https://github.com/gab1one/imglib2-bittype/blob/master/src/test/java/org/knime/knip/ArrayImgOversizeTest.java Like this one?

Comment (ctrueden) [reply src/main/java/net/imglib2/type/logic/BitType.java]: Something like that, yeah! Makes sense to add it here, right?

File: src/test/java/net/imglib2/type/logic/BitTypeArrayImgSizeTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+public class BitTypeArrayImgSizeTest
+{
+
+	@Test
+	public void testOverSizedBitImageExplicitStorageArray()
+	{
+
+		final int numLongs = ( int ) Math.ceil( 1000 * 1000 / 64 );
+
+		final ArrayImg< BitType, LongArray > img = ArrayImgs.bits( new LongArray( new long[ numLongs ] ), 1000, 1000 );
```
Comment (imagejan) [inline src/test/java/net/imglib2/type/logic/BitTypeArrayImgSizeTest.java]: Would it make sense to also test with image sizes that are not divisible by 64? That is, `1000 * 1000 / 64 == 15625`, so we don't actually test "partially filled" `long`s. How about a 100 * 100 image (`1000 * 1000 / 64 == 156.25`)?

Comment (gab1one) [reply src/test/java/net/imglib2/type/logic/BitTypeArrayImgSizeTest.java]: Good point @imagejan I modified the test :+1:

**Comment by tpietzsch** [comment]
Good catch @gab1one! Please format with imglib2 code style. Then it's good to go from my point of view.

**Pull request merged**